tests/unit/models/governing-body-test.js

```diff
@@ -0,0 +1,828 @@
+        },
+      });
+      assert.notPropContains(model.error, {
+        endDate: {
+          message: 'Kies een einddatum die na de startdatum plaatsvindt',
+        },
+      });
+    });
+
+    test('it returns a single error when the given end date is invalid', async function (assert) {
+      const model = this.store().createRecord('governing-body', {
+        startDate: new Date('2025-01-01'),
+        endDate: 'Not a date',
+      });
+
+      const isValid = await model.validate();
+
+      assert.false(isValid);
+      assert.strictEqual(Object.keys(model.error).length, 1);
+      assert.propContains(model.error, {
+        endDate: { message: 'Vul de einddatum in' },
+      });
+      assert.notPropContains(model.error, {
+        startDate: {
+          message: 'Kies een startdatum die vóór de einddatum plaatsvindt',
+        },
+      });
+      assert.notPropContains(model.error, {
+        endDate: {
+          message: 'Kies een einddatum die na de startdatum plaatsvindt',
+        },
+      });
+    });
+
+    test('it returns no errors when valid dates are given', async function (assert) {
+      const model = this.store().createRecord('governing-body', {
+        startDate: new Date('2025-01-01'),
+        endDate: new Date('2026-01-01'),
+      });
+
+      const otherBodiesStub = sinon.stub(model, 'getOtherTimedGoverningBodies');
+      otherBodiesStub.resolves([]);
+
+      const isValid = await model.validate();
+
+      assert.true(isValid);
+      assert.notOk(model.error);
+
+      otherBodiesStub.restore();
+    });
+
+    test('it returns two errors when the start date is after the end date', async function (assert) {
+      const model = this.store().createRecord('governing-body', {
+        startDate: new Date('2025-01-01'),
+        endDate: new Date('2024-01-01'),
+      });
+
+      const isValid = await model.validate();
+
+      assert.false(isValid);
+      assert.strictEqual(Object.keys(model.error).length, 2);
+      assert.propContains(model.error, {
+        startDate: {
+          message: 'Kies een startdatum die vóór de einddatum plaatsvindt',
+        },
+        endDate: {
+          message: 'Kies een einddatum die na de startdatum plaatsvindt',
+        },
+      });
+    });
+
+    test('it returns two errors when the end date is before the start date', async function (assert) {
+      const model = this.store().createRecord('governing-body', {
+        startDate: new Date('2024-01-01'),
+        endDate: new Date('2023-01-01'),
+      });
+
+      const isValid = await model.validate();
+
+      assert.false(isValid);
+      assert.strictEqual(Object.keys(model.error).length, 2);
+      assert.propContains(model.error, {
+        startDate: {
+          message: 'Kies een startdatum die vóór de einddatum plaatsvindt',
+        },
+        endDate: {
+          message: 'Kies een einddatum die na de startdatum plaatsvindt',
+        },
+      });
+    });
```

Comment:
It's a duplicate of the previous one ;)

Reply:
I agree that these two tests are currently the same, but I would opt to keep them both. I think it would make more sense to show an error only for a field if it was changed, instead of just always showing both error messages as is the current behaviour (copied from how it was before.) Imo having both tests already there can be beneficial when making such changes in the future, although this is clearly debatable.